### ISSUE TITLE
Add support to all libraries.

### ISSUE THREAD

**Issue by aviyadeveloper** [google-map-react/google-map-react#922]
Current implementation only supports the `visualization` library (for some reason naming it as heatMapLibrary?). google maps has 4 libraries available: `visualization`, `geometry`, `drawing`, `places`.

It would be great to have support to all of them, so they are available on the `maps` object in the `onGoogleApiLoaded` method.

I propose a fix while still supporting the `heatMapLibrary` prop in this PR:
https://github.com/google-map-react/google-map-react/pull/921

**Comment by itsmichaeldiego** [comment]
@aviyadeveloper You could do that by yourself, like @Nisanth-Reddy pointed out:

```javascript
<GoogleMap
    bootstrapURLKeys={{
      libraries: ['places', 'geometry', 'visualization', 'heatmap'],
      key: MAP_KEY
    }}
/>
```

**Comment by itsmichaeldiego** [comment]
Merged and released in `2.1.0`, let me know if everything works fine for you folks!

**Comment by itsmichaeldiego** [comment]
Need to reopen this as I had to revert https://github.com/google-map-react/google-map-react/pull/921 which had a major bug.

Will wait for @aviyadeveloper to fix it properly without causing bugs!

**Comment by itsmichaeldiego** [comment]
@aviyadeveloper Merged in #946 and Released in version `2.1.4`, let me know if you encounter any problems!
